README.md

```diff
@@ -628,6 +631,26 @@ subject.send(5)
 5
 ```
 
+## Convenience
+
+## Optional.publisher
+
+`Optional.publisher` is a method version of [`Optional.Publisher.init`](https://developer.apple.com/documentation/swift/optional/publisher/3343960-init). It puts the type on equal footing with [`Result.publisher`](https://developer.apple.com/documentation/swift/result/3344716-publisher).
+
+```swift
+Optional(1)
+  .publisher
+  /* … */
+```
+
+instead of
+
+```swift
+Optional.Publisher(Optional(1))
+  .publisher
+  /* … */
+```
```

Comment:
Maybe like this? 

```suggestion
let number: Int? = 1
number.publisher
  /* … */

instead of

let number: Int? = 1
Optional.Publisher(number)
  /* … */
```